### PR TITLE
Create Forge::dropPrimaryKey()

### PR DESCRIPTION
**Pull request by sclubricants**
This PR creates a method to drop primary keys from a table.

This is the second part of https://github.com/codeigniter4/CodeIgniter4/pull/6430

This solves part of https://github.com/codeigniter4/CodeIgniter4/issues/4653

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [x] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Review by MGatner — APPROVED** [review]
Awesome. Looks good to me but let's get another review from a database maintainer.

**Comment by MGatner** [comment]
@sclubricants Related to my last comment, I am working on putting together a team for database reviews and issues: https://forum.codeigniter.com/showthread.php?tid=82920

Would you be willing to join this team? It wouldn't be much different than how things are now, but you might get more requests to look at database Pull Requests. Any participation is of course optional (we're all volunteers here). Your expertise would be very welcome!

**Comment by sclubricants** [comment]
Sure, be glad to help.

Im working on a PDO/ODBC driver as we speak. ODBC gets messy. I half way built one a while back but it needs a lot of work. Would be great to have it as part of CI. I use it a lot.

**Review by kenjis — APPROVED** [review]
LGTM!

**Pull request merged**
